runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -118,10 +118,21 @@
 	CLAN_CHAT_NAME(WidgetID.CLAN_CHAT_GROUP_ID, WidgetID.ClanChat.NAME),
 	CLAN_CHAT_OWNER(WidgetID.CLAN_CHAT_GROUP_ID, WidgetID.ClanChat.OWNER),
 
+	BANK_CONTAINER(WidgetID.BANK_GROUP_ID, WidgetID.Bank.BANK_CONTAINER),
+	BANK_INNER_CONTAINER(WidgetID.BANK_GROUP_ID, WidgetID.Bank.BANK_INNER_CONTAINER),
+	BANK_SETTINGS(WidgetID.BANK_GROUP_ID, WidgetID.Bank.BANK_SETTINGS),
+	BANK_SCROLLBAR(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SCROLLBAR),
+	BANK_BOTTOM_BAR(WidgetID.BANK_GROUP_ID, WidgetID.Bank.BOTTOM_BAR),
+	BANK_SEARCH_BUTTON_BACKGROUND(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SEARCH_BUTTON_BACKGROUND),
+	BANK_TAB_BAR(WidgetID.BANK_GROUP_ID, WidgetID.Bank.TAB_BAR),
+	BANK_SOMETHING(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SOMETHING),
```

Comment:
Maybe this BANK_SOMETHING should be removed when it is no longer needed and naming it is not useful